### PR TITLE
Update README with more info

### LINKED ISSUE #19: no-arc packages
### Problem
For non experts the concept: "no-arc" packages might be confusing.

### Suggested Improvement
Please add an extra line as to explain what is this in the readme or a link to the installable packages

### PR DESCRIPTION
**Pull request by martinRenou**
Help resolve #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +47,7 @@ dependencies:
   - numpy
   - matplotlib
 ```
+
+Only ``no-arch`` packages from ``conda-forge`` and ``emscripten-forge`` can be installed.
+- **How do I know if a package is ``no-arch`` on ``conda-forge``?** ``no-arch`` means that the package is OS-independent, usually pure-python packages are ``no-arch``. To check if your package is ``no-arch`` on ``conda-forge``, check if there is a "no-arch" entry in its feedstock repository. *e.g.* ``ipyleaflet`` is ``no-arch`` because [it specifies it here](https://github.com/conda-forge/ipyleaflet-feedstock/blob/main/recipe/meta.yaml#L14). If your package is not ``no-arch`` but is a pure Python package, then you should probably update the feedstock to turn your package into a ``no-arch`` one.
```
Comment (jtpio) [inline README.md:52]: Sometimes a user facing website could also be handy to find this information more easily.

For example with https://prefix.dev/channels/conda-forge/packages/ipyleaflet:

image

Comment (jtpio) [reply README.md:52]: or with `beta.mamba.pm`:

image

Comment (martinRenou) [reply README.md:52]: Indeed! Thanks. I will probably use your beta.mamba.pm screenshot, as I was already mentioning that website for available emscripten packages.

Comment (jtpio) [reply README.md:52]: Makes sense for consistency

**Pull request merged**